pkgs/top-level/all-packages.nix

```diff
@@ -27241,4 +27241,6 @@ in
   gpio-utils = callPackage ../os-specific/linux/kernel/gpio-utils.nix { };
 
   navidrome = callPackage ../servers/misc/navidrome {};
+
+  apple-color-emoji = callPackage ../data/fonts/apple-color-emoji {};
```

Comment:
Not an issue per se, but all-packages.nix once had a rough structure & sorting. I'd personally move it to the start of `### DATA` to the bulk of other `/data/*` stuff, sorted alphabetically. But this structure isn't required from what I know.